service/history/execution/history_builder_test.go

```diff
@@ -1276,6 +1276,253 @@ func (s *historyBuilderSuite) TestHistoryBuilderWorkflowExternalSignaled() {
 	}]
 	s.True(exists)
 }
+
+func (s *historyBuilderSuite) TestHistoryBuilderActivityTaskTimedOut() {
+	id := "historybuilder-workflow-id"
```

Comment:
Since you are reusing those constants for different tests, you can use const at the top of the file to declare those fields and avoiding repetitive declaration in each test functions.

Reply:
Nit: i'd prefer const instead of variable declaration since you are not changing it.

Reply:
imho both ways are fine, the counterargument to putting them up the top of the file is for very big packages, then they're sorta hard to track down as they're package-scoped

Reply:
This is consistent with the rest of the file so I'm going to leave it as is. I think I generally would agree with you though. This file is massive and I don't think it's worth changing it or introducing a new standard.

Reply:
Got it, thanks for explaining.